Doc/library/functions.rst

```diff
@@ -738,6 +739,7 @@ are always available.  They are listed here in alphabetical order.
    value of that attribute.  For example, ``getattr(x, 'foobar')`` is equivalent to
    ``x.foobar``.  If the named attribute does not exist, *default* is returned if
    provided, otherwise :exc:`AttributeError` is raised.
+   *name* need not be a Python identifier (see :func:`setattr`).
```

Comment:
and same here?

Reply:
Resolving on the same principle as we agreed for the doc of `delattr()`.